server/models/user.model.js

```diff
@@ -90,19 +90,32 @@ const UserSchema = new mongoose.Schema(
     activationDate: Date,
     address: addressSchema,
     banned: {
-      by: {
-        type: ObjectId,
-      },
-      date: {
-        type: Date,
-      },
       status: {
         type: Boolean,
         default: false,
       },
-      reason: {
-        type: [String],
-      },
+      info: [
```

Comment:
As suggested in the last comment:

```suggestion
      case: [
```